src/assets/scss/_resizable.scss

```diff
@@ -0,0 +1,17 @@
+.divider-hitbox {
+    cursor: col-resize;
+    align-self: stretch;
+    display: flex;
+    align-items: center;
+    padding: 0 1.5rem 0 0.4rem;
+    top: 50%;
+    color: #ccc;
+    font-size: 2em;
+    color: #bbb;
+    left: 2px;
+}
+
+.divider {
```

Comment:
This class seems to be removed now, no need to keep the rule in the CSS.

Reply:
fixed